ophyd/pv_positioner.py

```diff
@@ -77,20 +75,19 @@ class PVPositioner(Device, Positioner):
     done_value = 1
     put_complete = False
 
-    def __init__(self, prefix='', *, settle_time=0.05, limits=None, name=None,
-                 timeout=None, read_attrs=None, configuration_attrs=None,
-                 monitor_attrs=None, parent=None,
-                 **kwargs):
+    def __init__(self, prefix='', *, limits=None, name=None, read_attrs=None,
+                 configuration_attrs=None, monitor_attrs=None, parent=None,
+                 egu='', **kwargs):
         super().__init__(prefix=prefix, read_attrs=read_attrs,
                          configuration_attrs=configuration_attrs,
                          monitor_attrs=monitor_attrs,
-                         name=name, parent=parent, timeout=timeout, **kwargs)
+                         name=name, parent=parent, **kwargs)
 
         if self.__class__ is PVPositioner:
             raise TypeError('PVPositioner must be subclassed with the correct '
                             'signals set in the class definition.')
 
-        self.settle_time = float(settle_time)
+        self._egu = egu
```

Comment:
Multiple signals that could specify the units (or not) - setpoint/readback. 

What's the preferred logic here?